cli.js

```diff
@@ -3,10 +3,12 @@
 /**
  * Parsing:
  *
- * $ debuguy parse <source_dir>
- * $ debuguy parse <source_dir> <destination_dir>
- * $ debuguy parse -r <source_dir>
- * $ debuguy parse -r <source_dir> <destination_dir>
+ * $ debuguy parse <source_dir> [options]
+ * $ debuguy parse <source_dir> <destination_dir> [options]
```

Comment:
I want to discuss [options] sequence actually. :)
Do you agree we move [options] to the last arguments?
If so, I will change the logic of these two lines in app.js
      var sourceDir = args.r ? args.r : args._[1];
      var destDir = args.r ? args._[1] : args._[2];

-r won't affect sourceDir and destDir anymore.